tests/lca/imagebasedupgrade/mgmt/upgrade/tests/e2e-upgrade-test.go

```diff
@@ -723,3 +744,24 @@ func findInstalledCSV(expectedCSV string) bool {
 
 	return false
 }
+
+// ClusterVersionXYZ returns the cluster's OCP version.
+func ClusterVersionXYZ(clusterObj cluster.APIClientGetter) (string, error) {
+	clusterVersion, err := cluster.GetOCPClusterVersion(clusterObj)
+	if err != nil {
+		return "", err
+	}
+
+	if !clusterVersion.Exists() {
+		return "", fmt.Errorf("cluster version not found")
+	}
```

Comment:
This doesn't seem necessary. Did you need this?

Reply:
Removed.
Thanks.